patches/ses+0.14.3.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/node_modules/ses/dist/ses.cjs b/node_modules/ses/dist/ses.cjs
+index 121da57..107c4ca 100644
+--- a/node_modules/ses/dist/ses.cjs
++++ b/node_modules/ses/dist/ses.cjs
+@@ -9808,7 +9808,7 @@ markVirtualizedNativeFunction)
+           // of a debugging attempt. See the comment at top of file for an
+           // explanation.
+           // eslint-disable-next-line @endo/no-polymorphic-call
+-          console.log(`Removing ${subPath}`);
++          console.warn(`Removing ${subPath}`);
```

Comment:
I see this. I should prioritize fixing this generally.

Reply:
The patch is an expedient to prevent actual failures when stdout is captured.  When we upgrade SES and it fails to apply, we'll get a warning.

Reply:
Would it fail to apply tho? The fix will not touch these warnings.

Why would stdout cause failures?

Reply:
> Would it fail to apply tho? The fix will not touch these console statements.

`patch-packages` tracks the version number of the package you're patching.  It will warn when the version doesn't match the one named by the `patches/*+VERSION.patch` file.

> Why would stdout cause failures?

```sh
./my-ses-program-running-on-node16 > ./critical-file.txt
```

Oops, now your `critical-file.txt` starts with:

```
Removing intrinsics.Object.hasOwn
```

Reply:
oh you're right, these should go to `stderr` instead of `stdout`! For some reason I misread the patch, sorry.
@kriskowal Any reason these are `console.log` instead of `console.warn` in the first place?